sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -139,9 +139,48 @@ Applications must make sure that the Level-Zero handles themselves aren't used s
 Practically speaking, and taking into account that SYCL runtime takes ownership of the Level-Zero handles,
 the application should not attempt further direct use of those handles.
 
+## 5 Level-Zero additional functionality
+
+### 5.1 Free Memory query
+Level Zero contains a library for system resource management called Sysman.  
+Sysman allows programmers to query the amount of free, or unallocated, 
+memory in a device.  This extension provides a new device information
+descriptor specific to Level-Zero Backends that allows DPC++ programs to query
+the amount of free memory for a given device. The new descriptor is as follows: 
+
+``` C++
+namespace sycl{
+namespace ext {
+namespace oneapi{
+namespace level_zero {
+namespace info {
+namespace device {
+
+struct free_memory {
+    using return_type = size_t;
+};
+
+} // namespace device;
+} // namespace info
+} // namespace level_zero
+} // namespace oneapi
+} // namespace ext
+} // namespace sycl
+```
+
+The new struct ```free_memory``` is used in conjuction with the ```get_backend_info()``` method of the ```device``` class in SYCL 2020.  The query will return the number of bytes of free memory for that device.

```

Comment:
Suggest using a table here to describe each info descriptor because that will be easy to extend as we add more.  The description for each query can be shorter since we talk about their general usage in the introduction above.  For this one, I think the description can just be:

> Returns the number of bytes of free memory for the device.